### PR TITLE
Docs: README: Add GOPATH requirement to source location

### PR DESCRIPTION
**Pull request by grahamwhaley**
As we now have golang code in the tree, you must have the repo/code
in your GOPATH, or it won't build. Add that info to the Building
section.

Fixes: #749 

Signed-off-by: Graham Whaley <graham.whaley@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
This looks good. However, we probably need to mention that the minimum version of `go` we support is `1.7.1` (from https://github.com/01org/cc-oci-runtime/blob/master/versions.txt#L1).

Of course, since the package isn't buildable automatically by `go get`, it -- currently -- doesn't really matter what version of go is used. However, we should specify it prominently to avoid problems for the user when they do attempt a build.

That information probably belongs in the "Dependencies" section though.

**Comment by chavafg** [comment]
qa-failed

**Comment by jodh-intel** [comment]
@grahamwhaley - looks like you've confused pullapprove :smile:

**Comment by grahamwhaley** [comment]
and qa!!??
Good point on v1.7 - that did occur to me at some point, so I will put it in. Thus, v2 coming.. adding dnm label)

**Comment by chavafg** [comment]
qa-failed

**Comment by dlespiau** [comment]
Hum, have we broken something recently?

**Comment by dlespiau** [comment]
lgtm

**Comment by grahamwhaley** [comment]
OK, we didn't break qa - it was me - due to a repo shuffle around I ended up basing off a very old master. Will do a rebase and re-submit...

**Comment by chavafg** [comment]
qa-failed

**Comment by chavafg** [comment]
qa-failed

**Comment by chavafg** [comment]
qa-failed

**Comment by chavafg** [comment]
qa-passed

**Comment by GabyCT** [comment]
@grahamwhaley one of the ci machines got frozen so that is why I need it to retest this pr but in the last attempt everything worked well

**Comment by amshinde** [comment]
lgtm

**Comment by jodh-intel** [comment]
lgtm

File: README.rst [hunk 1]
```diff
@@ -182,9 +196,10 @@ installed on your system:
 
 - check
 - glib
+- golang - version 1.7 or above
```
Comment (devimc) [inline README.rst]: I think that we should also specify the minimum required version of all packages
```
glib - version 2.46.2 or above
....
```
or is enough with versions.txt ?

Comment (dlespiau) [inline README.rst]: It's not a bad idea, but should be a separate commit/PR.

Comment (devimc) [inline README.rst]: nice, I'll do that :)

**Comment by gorozco1** [comment]
lgtm

**Comment by dlespiau** [comment]
I had to re-add/remove the do-not-merge label for pullapprove to notice that the label wasn't there. Now it says the commit doesn't have a sob tag - I disagree. Then it's missing one lgtm from the doc team, it got that one right. Once again pullapprove fails to impress :/

**Comment by dlespiau** [comment]
@grahamwhaley Oh I have a theory why pullapprove may reject your sob tag: git author email isn't the same than the one in the tag.

**Comment by grahamwhaley** [comment]
@dlespiau ah, indeed it is not - I did spot that. I really need to change my git global default.
Also, having submitted a similar change to ciao docs, we might have a better go get cmdline that does not print the benign warning out - so, I'm going to add the dnm label back in, and will submit a new, hopefully final?, version soon. thx.

**Comment by grahamwhaley** [comment]
OK, let's try again. Author and SOB now match (never had to use --reset-author before...), and thanks to @markdryan we now have a go get command that does not print the warning (thus, paragraph about benign warning also removed).

**Comment by chavafg** [comment]
qa-passed

**Comment by sboeuf** [comment]
lgtm

**Comment by sboeuf** [comment]
@mltullis, @rcaballeromx, could you please take a look at that and give your approval if you're okay with this PR ?

File: README.rst [hunk 2]
```diff
@@ -174,6 +174,18 @@ See `the contributing page <https://github.com/01org/cc-oci-runtime/blob/master/
 Building
 --------
 
+Acquiring
+~~~~~~~~~
+
+As ``cc-oci-runtime`` contains some golang code, the source tree needs to live
+in your ``$GOPATH`` in order to build. The easiest way to get the codebase into
+the correct place is with the command::
```
Comment (rcaballeromx) [inline README.rst]: Please do not use :: use instead .. code-block:: language
In this case, the line above ends with "with the command:"
The next line is blank and the following then becomes:
.. code-block:: console

Please do not forget to add a blank line before the command and to indent it.

Comment (sboeuf) [inline README.rst]: @grahamwhaley please make the changes and let's get this PR merged ;)

Comment (grahamwhaley) [inline README.rst]: Nope. That should be a new PR, for consistency in the doc. There are 14 or so other :: uses in this file, and no other code-blocks (and no other code-blocks in the whole repo that I can see).
So, I think we should merge as is with ::, and do another Issue/PR if we wish to move all the ::'s to code-blocks (and also decide if we do that for the other .rst files in the tree, which we probably should as there are only 2 others).

Comment (rcaballeromx) [inline README.rst]: For the sake of stop blocking this change, I will compromise. The other instances you mentioned were done before I was involved in the review process. Moving forward, added content should follow the best practices regardless of how it was done in the past. I agree that a separate PR is needed to fix all the previous instances.

**Comment by sboeuf** [comment]
Thanks @rcaballeromx. Could you file an issue to track those needed changes and refer this issue in this PR ?

**Comment by rcaballeromx** [comment]
Sure @sboeuf. I have done so #763

**Comment by sboeuf** [comment]
Great thanks @rcaballeromx !
